enhancements/cluster-logging/multi-cluster-observability.md

```diff
@@ -0,0 +1,622 @@
+  - https://issues.redhat.com/browse/OBSDA-489
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Multi-Cluster Observability Addon
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Multi-Cluster Observability has been an integrated concept in Red Hat Advanced Cluster Management (RHACM) since its inception but only incorporates one of the core signals, namely metrics, to manage fleets of OpenShift Container Platform (OCP) based clusters (See [RHACM Multi-Cluster-Observability-Operator (MCO)](rhacm-multi-cluster-observability)). The underlying architecture of RHACM observability consists of a set of observability components to collect a dedicated set of OCP metrics, visualizing them and alerting on fleet-relevant events. It is an optional but closed circuit system applied to RHACM managed fleets without any points of extensibility.
+
+This enhancement proposal seeks to bring a unified approach to collect and forward logs and OTLP data from a fleet of OCP clusters based on the RHACM addon facility (See Open Cluster Management (OCM) [addon framework](ocm-addon-framework)) by enabling these signals events to land on third-party managed and centralized storage solutions (e.g. AWS Cloudwatch, Google Cloud Logging). The multi-cluster observability addon is an optional RHACM addon. It is a day two companion for MCO and does not necessarily share any resources/configuration with the latter. It provides a unified installation approach of required dependencies (e.g. operator subscriptions) and resources (custom resources, certificates, CA Bundles, configuration) on the managed clusters to collect and forward logs and traces. The addon's name is Multi Cluster Observability Addon (MCOA).
+
+## Motivation
+
+The main driver for the following work is to simplify and unify the installation of log and trace collection and forwarding on an RHACM managed fleet of OCP clusters. The core utility function of the addon is to install required operators (i.e. [Red Hat OpenShift Logging](ocp-cluster-logging-operator) and [Red Hat OpenShift distributed tracing data collection](opentelemetry-operator)), configure required custom
+resources (i.e. `Clusterlogging`, `ClusterLogForwarder`, `OpenTelemetryCollector`) and reconcile per-cluster-related companion resources (i.e. `Secrets`, `ConfigMaps` for per-cluster authentication and configuration). This enables centralized control of the fleet's logs and traces collection and forwarding capabilities.
+
+### User Stories
+
+* As a fleet administrator I want to install a homogeneous log collection and forwarding on any set of RHACM managed OCP clusters.
```

Comment:
is there any thought on how we can provide say :
- audit logging only for clusters will label env=prod
- infra logging for all clusters
We do not have this for MCO at the moment. But we introduced a mechanism while adding User Workload data ingestion into ACM which could be exploited. We have not asked to do this yet in metric world. However, I wonder if this is something which we are used to seeing for logging/tracing.

Does the per cluster configmap shown below capable of doing that?